src/App.vue

```diff
@@ -70,6 +88,7 @@ import { CertifyStatementResource } from '@/resources'
 
 // Enums
 import { EntityTypes, FilingCodes } from '@/enums'
+import { NameRequestStates } from './enums/nameRequestStates'
```

Comment:
Instead of this, you should add nameRequestStates to @/enums/index.html, and then you can import them in the line above this.

Reply:
added the enum to the index and cleaned up the import